integration-box/kintone/main.dig

```diff
@@ -1,19 +1,19 @@
-timezone: Asia/Tokyo
-_export:
-  docker:
-    image: "digdag/digdag-python:3.7"
-  !include : 'config/kintone.yml'
-  org: hogehoge
-
-+get:
-  call>: get_records.dig
-
-+post:
-  call>: post_records.dig
-
-+put:
-  call>: put_records.dig
-
-+delete:
-  call>: delete_records.dig
-
+timezone: Asia/Tokyo
+_export:
+  docker:
+    image: "digdag/digdag-python:3.9"
+  !include : 'config/kintone.yml'
+  org: hogehoge
+
++get:
+  call>: get_records.dig
+
++post:
+  call>: post_records.dig
+
++put:
+  call>: put_records.dig
+
++delete:
+  call>: delete_records.dig
+
```

Comment:
This file changes file format from DOS to UNIX because almost all files in this repository use the UNIX file format.